nginx/conf.d/php_backend.conf

```diff
@@ -0,0 +1,12 @@
+## php backend settings
+fastcgi_pass    127.0.0.1:9000; #swap :9000 with :$port_switch when configuring hhvm/php-fpm routes;
+fastcgi_index   index.php;
+include         fastcgi_params;
+
+## specific security and compatibility headers
+add_header X-Frame-Options SAMEORIGIN;
+add_header X-Content-Type-Options nosniff;
+add_header X-XSS-Protection "1; mode=block";
```

Comment:
this needs to be disabled for IE8, where it can result in vulnerability with this header. It needs to be set to 0 for IE8